linera-sdk/src/ethereum.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+// Ethereum support is disabled on SDK v0.11
+#![cfg(not(target_arch = "wasm32"))]
+
```

Comment:
(For 0.12 we probably want to enable it? This branch already has the version set to 0.12.)

Reply:
Yep, it should be enabled already (it also fixes the issue on this branch).